lightning/src/routing/scoring.rs

```diff
@@ -1695,13 +1722,14 @@ impl<T: Time> Readable for ChannelLiquidity<T> {
 			min_liquidity_offset_history: min_liquidity_offset_history.unwrap(),
 			max_liquidity_offset_history: max_liquidity_offset_history.unwrap(),
 			last_updated,
+			decay_params: ProbabilisticScoringDecayParameters::default() // Unsure how to change the TLV here. lets use default first
```

Comment:
Not sure about the TLV read / write here. might need some help in understanding the scope here. currently its set to just reuse the default values upon reinit, which probably should be an okay decision